openstack/loadbalancer/v2/providers/results.go

```diff
@@ -0,0 +1,71 @@
+package providers
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Provider is the Octavia driver that implements the load balancing mechanism
+type Provider struct {
```

Comment:
I've updated the PR description to include https://github.com/openstack/octavia/blob/4a5c24ef6fcd3b5f198a20d780dedd7f7976296d/octavia/api/v2/types/provider.py#L20

Reply:
thanks!